src/Brancho/Resolver/JiraResolver.php

```diff
@@ -87,6 +91,38 @@ public function resolve(InputInterface $input, OutputInterface $output, ContextI
         return (array)$this->createBranchName($issue, $summary);
     }
 
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     * @param string $issue
+     * @param string $summary
+     *
+     * @return array|null
+     */
+    protected function createBranchFromIssueWithouParent(
+        InputInterface $input,
+        OutputInterface $output,
+        string $issue,
+        string $summary
+    ): ?array {
+        $question = new ConfirmationQuestion(
+            'Ticket without parent or epic branch. Proceed the creation? ' .
```

Comment:
On second thought, would it be more appropriate to simply inform the user that the issue has no parent and continue without requiring user input?

Reply:
I think so because by default it asks for confirmation if the branch may be created showing the name that the branch will have.. so I believe that just a warning is enough

Reply:
I already changed that.. I don't know why it's still showing the old code, but you can check that here: https://github.com/brunoviana-spryker/brancho/blob/bugfix/pbc-991-brancho-fails-when-using-it-with-a-ticket-that-does-not-have-an-epic-assigned/src/Brancho/Resolver/JiraResolver.php